dashboard/src/components/Header/Header.tsx

```diff
@@ -70,6 +68,7 @@ class Header extends React.Component<IHeaderProps, IHeaderState> {
     const reposPath = `/config/ns/${cluster.currentNamespace}/repos`;
     return (
       <section className="gradient-135-brand type-color-reverse type-color-reverse-anchor-reset">
+        <UISelector UI={featureFlags.ui} />
```

Comment:
It's not clear to me here - does this actually make a selector appear in the UI? I assume not since we've got a feature flag to select which UI... so what does this component do?

Reply:
Nope, I called this "selector" because the component is in charge of selecting the css file to load (but maybe another name is more suitable?). The selection is made automatically based on the configuration so the user just sees the configured UI. You will see an example in the next PR.